vault/identity_store_aliases.go

```diff
@@ -373,6 +403,56 @@ func (i *IdentityStore) handleAliasUpdate(ctx context.Context, req *logical.Requ
 	}, nil
 }
 
+//TODO: ask if we can use the same validation code from kv secrets plugin
```

Comment:
We can't for now. We can retain this function here.

Reply:
done